images/busybox/wolfi.tf

```diff
@@ -0,0 +1,27 @@
+module "latest-wolfi" {
+  source = "../../tflib/publisher"
+
+  target_repository = var.target_repository
+  config            = file("${path.module}/configs/latest.wolfi.apko.yaml")
+}
+
+module "version-tags-wolfi" {
+  source  = "../../tflib/version-tags"
+  package = "busybox"
+  config  = module.latest-wolfi.config
+}
+
+module "test-latest-wolfi" {
+  source = "./tests"
+  digest = module.latest-wolfi.image_ref
+}
+
+module "tagger-wolfi" {
+  source = "../../tflib/tagger"
+
+  depends_on = [module.test-latest-wolfi]
+
+  tags = merge(
+    { for t in toset(concat(["latest"], module.version-tags-wolfi.tag_list)) : t => module.latest-wolfi.image_ref },
```

Comment:
This should be `latest-glibc`, and `glibc-1.36.1-r0` for example.

Reply:
Good catch!

This makes me think that these should use the same tagger module invocation like other images to avoid the pitfall of racing tags like we inadvertently got here.